tdrs-backend/tdpservice/settings/common.py

```diff
@@ -273,6 +273,7 @@ class Common(Configuration):
     # of API POST calls to prevent false negative authorization errors.
     # https://docs.djangoproject.com/en/2.2/ref/settings/#csrf-cookie-httponly
     CSRF_COOKIE_HTTPONLY = False
+    CSRF_TRUSTED_ORIGINS = ['.app.cloud.gov']
```

Comment:
We needed to add this so the app would work on different backends, we were getting a 403 error otherwise.